Dangerfile.swift

```diff
@@ -26,4 +25,11 @@ let editedFiles = danger.git.modifiedFiles + danger.git.createdFiles
 let editedAppFiles = editedFiles.filter { $0.contains("/App") }
 message("Files:-\(editedFiles)")
 
+
```

Comment:
<!--
  0 failure: 
  1 warning:  Limit vertical wh...
  
  
  DangerID: danger-id-Danger;
  File: Dangerfile.swift;
  Line: 28;
-->

- :warning: Limit vertical whitespace to a single empty line. Currently 2. (`vertical_whitespace`)